backend/metamodel/syson-sysml-metamodel/src/test/java/org/eclipse/syson/sysml/util/ModelBuilder.java

```diff
@@ -0,0 +1,134 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.syson.sysml.util;
+
+import static java.util.stream.Collectors.toMap;
+
+import java.util.Map;
+
+import org.eclipse.emf.ecore.EClass;
+import org.eclipse.syson.sysml.Classifier;
+import org.eclipse.syson.sysml.Element;
+import org.eclipse.syson.sysml.Feature;
+import org.eclipse.syson.sysml.FeatureMembership;
+import org.eclipse.syson.sysml.FeatureTyping;
+import org.eclipse.syson.sysml.OwningMembership;
+import org.eclipse.syson.sysml.Relationship;
+import org.eclipse.syson.sysml.Subclassification;
+import org.eclipse.syson.sysml.SysmlFactory;
+import org.eclipse.syson.sysml.SysmlPackage;
+import org.eclipse.syson.sysml.Type;
+import org.eclipse.syson.sysml.VisibilityKind;
+import org.eclipse.syson.sysml.helper.EMFUtils;
+
+/**
+ * Helper used to build SysML model.
+ * 
+ * @author Arthur Daussy
+ */
+public class ModelBuilder {
+
+    private static final Map<Class<?>, EClass> CLASS_TO_ECLASS;
+
+    private final SysmlFactory fact = SysmlFactory.eINSTANCE;
+
+    static {
+        CLASS_TO_ECLASS = EMFUtils.eAllContentSteamWithSelf(SysmlPackage.eINSTANCE).filter(e -> e instanceof EClass eClass && !eClass.isAbstract() && !eClass.isInterface()).map(EClass.class::cast)
```

Comment:
For your information, an adapter already exists and do the same thing.
Please see `SysONEContentAdapter`, `SysMLEditingContextProcessor` where `SysONEContentAdapter` is instantiated, and `UtilService#getAllReachable` for an exemple of use.

Reply:
I'm not sure it is exactly the same thing. 
My map is Map<Class<?>, EClass>. It only purpose is to be able to create new instances by using java classes.
It does not keep track of the instance of a type.
Or I missing something

Reply:
Ok, I see.
In this case, may be `SysMLMetamodelHelper` would fit your needs.

Reply:
I will accept this PR anyway.